docs/explanations/modeling_extensions/index.rst

```diff
@@ -17,42 +17,55 @@ provided below.
     uncertainty_propagation/index
     diagnostics/index
 
+.. rubric:: PySMO: Python-based Surrogate Modeling Objects
+
+:ref:`PySMO: Python-based Surrogate Modeling Objects<explanations/modeling_extensions/surrogate/api/pysmo/index:PySMO: Python-based Surrogate Modeling Objects>`
+provides tools for generating different types of reduced order models. It provides IDAES users 
+with a set of surrogate modeling tools which supports flowsheeting and direct integration into 
+an equation-oriented modeling framework. It allows users to directly integrate reduced order 
+models with algebraic high-fidelity process models within an single IDAES flowsheet.
+
+.. image:: /images/pysmo-logo.png
+    :width: 500px
+    :align: center
+
+.. rubric:: OMLT: Optimization and Machine Learning Toolkit
+
+:ref:`OMLT: Optimization and Machine Learning Toolkit<explanations/modeling_extensions/surrogate/api/omlt-keras/index:OMLT: Optimization and Machine Learning Toolkit>`
+provides tools for generating machine learning, neural network and gradient-boosted tree models with the Pyomo optimization environment. It provides IDAES users optimization formulations for ML models (e.g. full-space, reduced-space, MILP) and an interface to import Sequential Keras and general ONNX models.
+
+.. image:: /images/omlt-keras-logo.png
+    :width: 500px
+    :align: center
+
 .. rubric:: ALAMOPY: ALAMO Python
 
-:ref:`ALAMOPY<explanations/modeling_extensions/surrogate/alamopy/index:ALAMOPY: ALAMO Python>`
+:ref:`ALAMOPY: ALAMO Python<explanations/modeling_extensions/surrogate/api/alamopy/index:ALAMOPY: ALAMO Python>`
 provides a wrapper for the software ALAMO which generates algebraic surrogate models of 
```

Comment:
We should probably add a note block here to say that ALAMO is an external tool that needs to be installed separately as well.